scripts/mdpow-solvationenergy

```diff
@@ -185,31 +226,48 @@ if __name__ == "__main__":
+    parser.add_option('-o', '--outfile', dest="outfile",
+                      help="append one-line results summary to FILE [%default]",
                       metavar="FILE")
     parser.add_option('-e', '--energies', dest="energyfile",
                       help="append solvation free energies to FILE [%default]",
                       metavar="FILE")
+    parser.add_option('--estimator', dest="estimator", default='alchemlyb',
+                      help="choose the estimator to be used, alchemlyb or mdpow estimators")
+    parser.add_option('--method', dest="method", default='MBAR',
+                      help="choose the method to calculate free energy",)
     parser.add_option('--force', dest='force',
                       action="store_true",
                       help="force rereading all data [%default]")
+    parser.add_option('--noSI', dest='SI',
+                      action="store_false",
+                      help="Disable statistical inefficiency analysis"
+                      "Statitical inefficiency analysis is performed by default when using"
+                      "alchemlyb estimators and is disabled when using mdpow estimator.")
     parser.add_option('-s', '--stride', dest="stride", type='int',
                       help="Use every N-th datapoint from the original dV/dlambda data. "
-                      "Using a number larger than 1 can significantly reduce memory usage "
-                      "with little impact on the precision of the final output. [%default]",
+                      "[%default]",
                       metavar="N")
-    parser.add_option('--solvent', '-S', dest='solvent',
-                      metavar="NAME",
-                      help="solvent NAME for compound, 'water', 'octanol', or 'cyclohexane' [%default]")
+    parser.add_option('--start', dest="start", type='int',
+                      help="Start point for the data used from the original dV/dlambda data. ")
+    parser.add_option('--stop', dest="stop", type='int',
+                      help="Stop point for the data used from the original dV/dlambda data.",)
     parser.add_option('--ignore-corrupted', dest="permissive",
                       action="store_true",
                       help="skip lines in the md.xvg files that cannot be parsed. "
                       "WARNING: Other lines in the file might have been corrupted in "
                       "such a way that they appear correct but are in fact wrong. "
                       "WRONG RESULTS CAN OCCUR! USE AT YOUR OWN RISK [%default]")
-    parser.set_defaults(plotfile="auto",
-                        energyfile="gsolv.txt",
-                        force=False, stride=10, permissive=False, solvent='water')
+    parser.set_defaults(plotfile="none",
+                        outfile="pow.txt", energyfile="energies.txt",
+                        force=False, stride=1, permissive=False, SI=True)
```

Comment:
Let's replace the parser with argparse and put the defaults with the add_option calls.